locale/de.json

```diff
@@ -22,7 +20,24 @@
 		"global.social-pinterest": "Pinterest",
 		"global.social-email": "E-Mail",
 		"global.opens-in-new-window": "(Wird in einem neuen Fenster geöffnet)",
-		"global.sponsored-content": "Werblicher Inhalt"
+		"global.sponsored-content": "Werblicher Inhalt",
+		"global.gallery-text": "Gallery",
+		"global.video-text": "Video",
+		"global.social-email-content": "Send an email to %{authorName}",
+		"global.social-rss": "Subscribe to %{authorName} RSS feed",
+		"global.social-soundcloud-content": "Listen to %{authorName} on SoundCloud",
```

Comment:
We've now lost our translations?

Reply:
it looks like it? I'm not sure if these have been migrated in @malavikakoppula 's pr

Reply:
maybe the keys there point to the correct ones

Reply:
fixed https://github.com/WPMedia/arc-themes-blocks/pull/1499